ui/src/main/java/edu/wpi/grip/ui/pipeline/PipelineView.java

```diff
@@ -154,10 +154,12 @@ private OutputSocketView findOutputSocketView(OutputSocket socket) {
     }
 
     /**
+     * This function is public because it is used by {@link edu.wpi.grip.ui.preview.PreviewsView#onSocketPreviewChanged}
+     * to order the list of previews.
+     *
```

Comment:
This comment isn't really necessary and, as @ThomasJClark said before, Intellij does this for you by default.